tests/io/v3/base/test_csv_iterator.py

```diff
@@ -74,3 +123,19 @@ def test_csv_iterator(api):
     )
     assert next(csv_iterator) == CSV_TEXT
     assert next(csv_iterator) == CSV_TEXT_2
```

Comment:
here we would need to register CSV_TEXT_2 in the body as well correct? while registering the response
CC: @milanaghera-tenable

Reply:
No, this csv_text_2 is being handled by the iterator itself.